llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -526,17 +615,17 @@ Type *SPIRVToLLVM::transTypeWithOpcode<OpTypePointer>(SPIRVType *const spvType,
       if (!imagePtrTy)
         return samplerPtrTy;
       return StructType::get(*m_context, {imagePtrTy, samplerPtrTy});
-    }
-#if VKI_RAY_TRACING
-    else if (spvElementType->isTypeAccelerationStructureKHR()) {
+    } else if (spvElementType->isTypeAccelerationStructureKHR()) {
       pointeeLayout = LayoutMode::Explicit;
       // From now on (GPURT major version >= 34), AS header may start at a non-zero offset, GPURT now request base
       // offset of the resource, and it will calculate the actual GPUVA, instead of compiler providing one loaded from
       // offset 0. Here we use SPIRAS_Constant because later in llpcSpirvLowerGlobal the AS will be lowered to
       // get.desc.ptr which returns SPIRAS_Constant ptr.
       addrSpace = SPIRAS_Constant;
+    } else {
+      // Uniform constant variable outside of a block use std430 layout.
+      pointeeLayout = isAccelerationStructureType(spvElementType) ? LayoutMode::Explicit : LayoutMode::Std430;
```

Comment:
Merge this `else if` with `else`
```suggestion
    } else {
      // Uniform contant variable outside of a block use std430 layout.
      pointeeLayout = isAccelerationStructureType(spvElementType) ? LayoutMode::Explicit : LayoutMode::Std430;
      // From now on (GPURT major version >= 34), AS header may start at a non-zero offset, GPURT now request base
      // offset of the resource, and it will calculate the actual GPUVA, instead of compiler providing one loaded from
      // offset 0. Here we use SPIRAS_Constant because later in llpcSpirvLowerGlobal the AS will be lowered to
      // get.desc.ptr which returns SPIRAS_Constant ptr.
      addrSpace = isAccelerationStructureType(spvElementType) ? SPIRAS_Constant : addrSpace;
```